core/cmd/remote_client_test.go

```diff
@@ -694,3 +694,20 @@ func TestClient_CreateExtraKey(t *testing.T) {
 
 	assert.NoError(t, client.CreateExtraKey(c))
 }
+
+func TestClient_SetMinimumGasPrice(t *testing.T) {
+	t.Parallel()
+
+	app, cleanup, _ := setupWithdrawalsApplication(t)
+	defer cleanup()
+
+	assert.NoError(t, app.StartAndConnect())
+
+	client, _ := app.NewClientAndRenderer()
+	set := flag.NewFlagSet("setgasprice", 0)
+	set.Parse([]string{"8616460799"})
```

Comment:
Because of all the logic around supporting decimal gwei values, like `20.5`, could we have a test cover that as well? Would be great if we could assert the final value at the end too:

```
assert.Equal(t, big.NewInt(2050000000), app.Config.EthGasPriceDefault())
```